.github/workflows/build.yaml

```diff
@@ -142,7 +146,7 @@ jobs:
         run: bin/build-darwin.sh
 
   build_and_test_bmqprometheus_plugin:
-    name: "Check if BMQPrometheus plugin can build and pass tests"
+    name: "Build plugin [ubuntu]"
```

Comment:
Maybe having `Prometheus` here is good, or if this job is meant for all future plugins as well, `Build plugins [ubuntu]`.